Shuffle.cs

```diff
@@ -475,61 +475,52 @@ private void SetTatlColour()
             };
         }
 
-        private void MakeSpoilerLog()
+        private void CreateTextSpoilerLog(Spoiler spoiler, string path)
```

Comment:
I would like the text spoiler log to also be done using a template. Something to do later i suppose.